snorkel/slicing/modules/slice_combiner.py

```diff
@@ -6,7 +6,22 @@
 
 
 class SliceCombinerModule(nn.Module):
-    """A module for combining the weighted representations learned by slices"""
+    """A module for combining the weighted representations learned by slices.
+
+    Intended for use with task flow including:
+        * Indicator operations
+        * Prediction operations
+        * Prediction transform features
+
+    Parameters
+    ----------
+    slice_ind_key : optional
```

Comment:
rm optional

Reply:
how come? there are default parameters?

Reply:
anything that's also specified by the type hints (besides parameter name) should go, since Sphinx will parse it for us and we don't want to maintain it in two places. this is also imprecise, since `typing.Optional` means something else.

Reply:
guessing it was added by autodocstring, so we should see if we can disable